core/js/src/main/scala/zio/internal/Sync.scala

```diff
@@ -0,0 +1,8 @@
+package zio.internal
+
+object Sync {
+  def apply[A](anyRef: AnyRef)(f: => A): A = {
```

Comment:
For `STM.scala` to be shared, this method should take the same arguments as `Sync` in `jvm` does.
But the compiler does not allow us to take `anyRef` and does nothing with it.
So I avoided the error just by asserting the existence of `anyRef`.

Reply:
You can do, inside the body:

```scala
val _ = anyRef
```

Reply:
I learned another thing! Thank you so much :)